### ISSUE TITLE
Organization has no public members

### ISSUE THREAD

**Issue by markulf** [mitls/mitls-f7#2]
Should we add everyone from the miTLS team as members?

**Comment by beurdouche** [comment]
Hi Markulf,

I think this is a temp repository that we should not advertise yet, but Alfredo should confirm that …

B.

**Comment by prosecco** [comment]
Yes, let’s add all milts team members, but please do not advertise the repo yet.
We need to do a full code review, which we should schedule soon.

-K.
